main.go

```diff
@@ -1,48 +1,84 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"log"
 	"os"
 
 	"github.com/SUSE/scf-secret-generator/secrets"
 )
 
-func printHelp() {
-	fmt.Printf("Usage: %s <role-manifest>\n", os.Args[0])
-}
+var domain = flag.String(
+	"domain",
+	"",
+	"Domain",
+)
+
+var namespace = flag.String(
+	"namespace",
+	"",
+	"Kubernetes namespace",
+)
+
+var serviceDomainSuffix = flag.String(
+	"serviceDomainSuffix",
+	"",
+	"Service Domain Suffix",
+)
+
+var secretsName = flag.String(
+	"secretsName",
+	"",
+	"Secrets Name",
+)
+
+var secretsGeneration = flag.String(
+	"secretsGeneration",
+	"",
+	"Secrets Generation",
```

Comment:
Could use a clearer explanation here

Reply:
I added some text, but this tool is only ever called from a script with the parameters filled in via ERB templating, so I doubt anyone ever sees the text...